src/impl/tpls/KokkosBlas_Host_tpl.cpp

```diff
@@ -812,6 +861,15 @@ namespace KokkosBlas {
                      (const std::complex<double>*)a, &lda,
                      (      std::complex<double>*)b, &ldb);
     }
+    template<>
+    void 
+    HostBlas<std::complex<double> >::gesv(int n, int rhs,
+                                         std::complex<double> *a, int lda, int *ipiv,
```

Comment:
I do find it weird that `HostBlas` is templated on `std::complex` but the instantiations use `Kokkos::complex`.

Reply:
Thanks, @mhoemmen . See my comment above.